src/PlayFabBuddy.Cli/Commands/Player/DeletePlayersBySegmentCommandSettings.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System.ComponentModel;
+using PlayFabBuddy.Lib.Interfaces.Adapter;
+using Spectre.Console.Cli;
+
+namespace PlayFabBuddy.Cli.Commands.Player;
+
+public class DeletePlayersBySegmentCommandSettings : PlayerSettings
+{
+    [Description(
+        $"The name of the segment Accounts should be deleted from. Use \"{IPlayStreamAdapter.AllPlayersSegmentName}\" to delete all Players.")]
+    [CommandArgument(0, "<SegmentName>")]
+    public string SegmentName { get; init; } = "";
+    
+    [Description("Prints additional information, like the Master Player Account IDs of the deleted accounts")]
```

Comment:
Is this a good place for this parameter? Maybe it would be better to have this in the `PlayerSetting` or even a new `GeneralCommandSettings` as it in theory could be applied to more commands?

Reply:
Yes, was my thought. But I would make this a separate issue and leave it here for now. I think we would need to come up with a proper idea of what "verbose" means and how we want this handled.